src/metabase/driver.clj

```diff
@@ -971,3 +974,12 @@
   {:added "0.48.0", :arglists '([driver database & args])}
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
+
+(defmulti schema+table->row-count
+  "Returns a map of [schema, table name] -> the estimated row count of all tables in the current database.
```

Comment:
nitpick: "the estimated row count of all tables ..." sounds like a summation over every table, how about:

> Returns a map of [schema, table name] -> the estimated row count, for each table in the current database.

Reply:
[81087df](https://github.com/metabase/metabase/pull/39494/commits/81087df06972202cf006c1f21214e18071bc7f24)